engines/dreamweb/use.cpp

```diff
@@ -179,6 +185,14 @@ void DreamWebEngine::showFirstUse() {
 	findNextColon(&obText);
 	findNextColon(&obText);
 	useText(obText);
+
+	if (_ttsMan != nullptr && ConfMan.getBool("tts_enabled_objects")) {
+		const char *text = (const char *)obText;
+		const char *colon_pos = strchr(text, ':');
+		Common::String result(text, colon_pos ? colon_pos - text : strlen(text));
+		_ttsMan->say(result, _textEncoding);
+	}
```

Comment:
You can use `textToSpeech()` here

Reply:
Indeed.
And I would also suggest renaming `textToSpeech()` into something like `speakObjectName()` to clarify that it cannot be used to speak conversation text.